Node/nextjs/__tests__/index.test.js

```diff
@@ -0,0 +1,80 @@
+import 'jest-canvas-mock'
+import "@testing-library/jest-dom";
+import Home from '../src/pages/index'
+import { fireEvent, render, screen, waitFor, act, cleanup } from "@testing-library/react";
+import axeDevtools from '@axe-devtools/browser'
+import Reporter from '@axe-devtools/reporter'
+import rimraf from 'rimraf'
+
+describe('@axe-devtools/browser, jest, @testing-library/react', () => {
+  let reporter
+
+  beforeAll(() => {
+    // https://github.com/dequelabs/axe-core/blob/develop/README.md?plain=1#L83
+    axeDevtools.configure({
+      rules: [
+        {
+          id: 'color-contrast',
+          enabled: false
+        }
+      ]
+    })
+    rimraf.sync('./a11y-results*')
+    reporter = new Reporter('a11y-results', './a11y-results')
+  })
+
+  afterAll(() => {
+    reporter.buildHTML('./a11y-results')
+    reporter.buildCSV('./a11y-results')
+    reporter.buildJUnitXML('./a11y-results')
+    cleanup()
+  })
+
+  it('ToDo App Home Page', async () => {
+    const { container } = render(<Home />)
+    
+    //Accessibility scan with axe DevTools API
+    const results = await axeDevtools.run(container)
+    reporter.logTestResult('home-page', results)
+  })
+
+  it("Add ToDo", async () => {
```

Comment:
nit - same here, test should describe what is being tested, example:
```suggestion
describe('when adding a new todo item, () => {
  it("should append onto the end of the list", async () => {
  // ... 
  })
})
```